starknet_py/net/full_node_client.py

```diff
@@ -387,22 +431,22 @@ async def get_contract_nonce(
 def get_block_identifier(
     block_hash: Optional[Union[Hash, Tag]] = None,
     block_number: Optional[Union[int, Tag]] = None,
-) -> dict:
+) -> Union[Dict, Tag, int, Hash, None]:
```

Comment:
I prefer to return the dict as it was earlier. Without that, we have to write `{"block_id": ...}` everywhere, which seems unreasonable.